### PR TITLE
KAA-1669: Jira flow - new page. [master]

### PR DESCRIPTION
**Pull request by hunch7**

### CODE REVIEW

File: doc/Customization-guide/How-to-contribute/Jira-flow/index.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+{:toc}
+
+{% include variables.md %}
+
+The Kaa project team uses [Atlassian Jira](https://www.atlassian.com/software/jira) for issue tracking in the development process.
+This section describes when and how you can create Jira issues at [http://jira.kaaproject.org/](http://jira.kaaproject.org/) if you decided to contribute to Kaa project.
+
+## When to create issues
+
+To decide whether you should create a Jira issue, use [Kaa Stack Overflow](http://stackoverflow.com/questions/tagged/kaa) to consult with Kaa team and find out whether your issue:
```
Comment (rasendubi) [inline doc/Customization-guide/How-to-contribute/Jira-flow/index.md]: Well, that's not what Stack Overflow is for. Users are expected to create Jira issues whenever they believe they have found a bug or have a feature request. Though, they should search the Jira to find out if the issue already exists first.

File: doc/Customization-guide/How-to-contribute/Jira-flow/index.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+|-------|-------------|
+| Project | Select the project from the drop-down list depending on which [Kaa project repository](https://github.com/kaaproject) the issue is related to. When in doubt, select **Kaa**. |
+| Issue Type | See [Issue types](#issue-types) to make the right choice. When in doubt, select **Task**. |
+| Summary | Provide a one-line summary of the issue in clear and concise language. |
+| Component/s | The drop-down list of this field depends on the project you selected in the **Project** field. |
+| Affects Version/s | Select the version affected by the issue. |
+| Description | Provide a meaningful description of the issue with all the details you find relevant. Describe the steps to reproduce the issue, and the environment in which you found it. In general, feel free to share any information you think is related to the issue. See also [Issue types](#issue-types) for specific information to be provided depending on the issue type. |
+| Attachment | Feel free to attach images, documents and other files you think can help understand the issue-related information. |
+
+>**NOTES:** Please do not fill any other fields, they will be managed by the Kaa team members.
```
Comment (rasendubi) [inline doc/Customization-guide/How-to-contribute/Jira-flow/index.md:33]: Why not forbid that on permission level?

File: doc/Customization-guide/How-to-contribute/Jira-flow/index.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+Bugs are usually code or documentation deficiencies.
+As a minimum, a Bug must contain the description of the problem encountered, steps to reproduce, version of the code tested, and provide the corresponding logs.
+Useful additions may include a reference to the test scenario, links to the documentation that show the expected system behavior, etc.
+It is allowed to check-in code directly in the Bug type issues.
+A Bug may be filed under an Epic if the affected functionality was related to the scope of that Epic.
+
+## Working on issues
+
+After you submitted an issue, it will appear in the selected project space for the Kaa team members to spot.
+The work on the issue can begin after the team lead included it in a sprint and a person is assigned to the issue.
```
Comment (rasendubi) [inline doc/Customization-guide/How-to-contribute/Jira-flow/index.md]: This section ignores the fact that any user can work on the issue (not team members only)

File: doc/Customization-guide/How-to-contribute/index.md [hunk 4]
```diff
@@ -18,15 +18,15 @@ We would love to see you mastering the Kaa source code; however, writing code is
 Feel free to use all of the contribution options:
 
 - Join [Kaa Stack Overflow](http://stackoverflow.com/questions/tagged/kaa) to find answers to common user questions, ask your own Kaa-related questions, and help others in the community.
-- Use [Jira](http://jira.kaaproject.org/browse/KAA/) to provide feedback, make features and functionality suggestions, and help other community members.
+- Use [Jira](http://jira.kaaproject.org/) to provide feedback, make features and functionality suggestions, and help other community members.
```
Comment (rasendubi) [inline doc/Customization-guide/How-to-contribute/index.md]: Add a link to Jira flow here, too. (As it is not limited to contributing to the source code.)

Comment (hunch7) [reply doc/Customization-guide/How-to-contribute/index.md]: Done.

File: doc/Customization-guide/How-to-contribute/Jira-flow/index.md (hunk 1 above)
Comment (hunch7) [reply doc/Customization-guide/How-to-contribute/Jira-flow/index.md]: Fixed.

File: doc/Customization-guide/How-to-contribute/Jira-flow/index.md (hunk 3 above)
Comment (hunch7) [reply doc/Customization-guide/How-to-contribute/Jira-flow/index.md]: Fixed.

File: doc/Customization-guide/How-to-contribute/Jira-flow/index.md (hunk 2 above)
Comment (hunch7) [reply doc/Customization-guide/How-to-contribute/Jira-flow/index.md:33]: Agreed. Will talk to Jira admin.

**Review by rasendubi — APPROVED** [review]

File: doc/Customization-guide/How-to-contribute/Jira-flow/index.md (hunk 2 above)
Comment (hunch7) [reply doc/Customization-guide/How-to-contribute/Jira-flow/index.md:33]: As discussed with Jira admin, these permissions will be configured after we migrate to Jira 7 which is planned to be done at the end of the current sprint.

File: doc/Customization-guide/How-to-contribute/Jira-flow/index.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+Useful additions may include a reference to the test scenario, links to the documentation that show the expected system behavior, etc.
+It is allowed to check-in code directly in the Bug type issues.
+A Bug may be filed under an Epic if the affected functionality was related to the scope of that Epic.
+
+## Sending error logs
+
+You may want to send some Kaa error logs to help resolve an issue.
+To do this:
+
+1. [Change the log level for the Kaa service to “TRACE”]({{root_url}}Administration-guide/Troubleshooting/#how-to-change-the-service-logging-level).
```
Comment (Polinaa) [inline doc/Customization-guide/How-to-contribute/Jira-flow/index.md]: Doesn't redirect to the specific section of the documentation.

File: doc/Customization-guide/How-to-contribute/Jira-flow/index.md [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+
+2. [Clear the Kaa logs]({{root_url}}Administration-guide/Troubleshooting/#how-to-clear-the-kaa-logs).
```
Comment (Polinaa) [inline doc/Customization-guide/How-to-contribute/Jira-flow/index.md]: the same

File: doc/Customization-guide/How-to-contribute/Jira-flow/index.md [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+
+3. Reproduce your issue.
+
+4. [Download the Kaa logs from the Sandbox]({{root_url}}Administration-guide/Troubleshooting/#how-to-download-the-kaa-logs-from-the-sandbox).
```
Comment (Polinaa) [inline doc/Customization-guide/How-to-contribute/Jira-flow/index.md]: the same

File: doc/Customization-guide/How-to-contribute/Jira-flow/index.md (hunk 5 above)
Comment (hunch7) [reply doc/Customization-guide/How-to-contribute/Jira-flow/index.md]: Fixed.

File: doc/Customization-guide/How-to-contribute/Jira-flow/index.md (hunk 6 above)
Comment (hunch7) [reply doc/Customization-guide/How-to-contribute/Jira-flow/index.md]: Fixed.

File: doc/Customization-guide/How-to-contribute/Jira-flow/index.md (hunk 7 above)
Comment (hunch7) [reply doc/Customization-guide/How-to-contribute/Jira-flow/index.md]: Fixed.

**Review by vadimol — APPROVED** [review]

**Pull request merged**